content/applications/hr/time_off.rst

```diff
@@ -465,52 +521,114 @@ If an employee has used all their time off, or is going to run out of time off,
 
-When the form is complete, click the :guilabel:`Save` button to save the information, and submit the
-request.
+If the request was created from the :guilabel:`dashboard`, click the :guilabel:`Save & Close` button
+to save the information and submit the request. If the form was completed from the :guilabel:`My
+Allocations` list view, the information is automatically saved as it is entered. However, the form
+can be saved manually at any time by clicking the :guilabel:`Save manually` option, represented by a
+:guilabel:`cloud with an upwards arrow inside` icon.
 
 .. image:: time_off/allocation-request.png
    :align: center
    :alt: An allocation request form filled out for an employee requesting an additional week of
          sick time.
 
-Approvals
-=========
+.. _time_off/approvals:
+
+Management
+==========
 
-Most requests for time off and allocations need to go through the approval process, prior to the
-time off being allocated, and then granted to an employee. Requests either need one or two
-approvals, depending on how the specific type of time off is configured.
+Most requests for time off and allocations need to go through an approval process prior to the time
+off being granted to an employee. Requests either need one or two approvals, if any, depending on
+how the specific type of time off is configured. These are all housed under the
+:guilabel:`Management` section of the *Time Off* application.
 
-Only users who can approve allocation and time off requests have the :guilabel:`Approvals` section
+Only users who can approve allocation and time off requests have the :guilabel:`Management` section
 visible in the *Time Off* application.
 
-Approve allocations
--------------------
+.. _time_off/manage-time-off:
+
+Manage time off
+---------------
+
+To view time off requests that need approval, navigate to :menuselection:`Time Off app -->
+Management --> Time Off`. The only time off requests visible on this list are for employees the user
+has either :guilabel:`Time Off Officer` or :guilabel:`Administrator` access rights for the *Time
+Off* application.
+
+The default filters in the :guilabel:`Time Off` list is :guilabel:`Waiting For Me`. This only
+presents time off requests that need to be approved for current employees on the user's team, with a
+status of either :guilabel:`To Approve` or :guilabel:`Second Approval`.
+
+The left side of the screen has various grouping options to narrow down the presented time off
+requests. Since only time off requests that need to be approved are shown, the only status options
+are :guilabel:`All`, :guilabel:`To Approve`, and :guilabel:`Second Approval`. The various
+:guilabel:`Departments` the user is a member of and manage employees under, also appears on the left
+side.
+
+To view requests with other statuses, first remove the :guilabel:`Waiting For Me` filter, by
+clicking the :guilabel:`✖️ (remove)` icon next to the :guilabel:`Waiting For Me` filter to remove
+it.
```

Comment:
Totally up to you, the writer's choice, but I personally don't think we need to be so granular in explaining that we click the x to close a filter. I personally think "remove the filter" is enough. Totally your opinion, so feel free to ignore!
```suggestion
To view requests with other statuses, first remove the :guilabel:`Waiting For Me` filter.
```